task_geo/data_sources/hdx_acap/hdx_acap_formatter.py

```diff
@@ -17,8 +19,13 @@ def hdx_acap_formatter(raw):
     data = raw.copy()
     data.columns = [column.lower() for column in data.columns]
     data = data.drop(['pcode', 'admin_level_name', 'alternative source'], axis=1)
-    column_order = ['id', 'country', 'region', 'iso', 'category', 'measure',
+    column_order = ['id', 'country', 'region', 'country_iso2', 'category', 'measure',
                     'targeted_pop_group', 'comments', 'non_compliance', 'date_implemented',
                     'source', 'source_type', 'entry_date', 'link']
+    # Convert ISO3 Country code to ISO2 Country Code
+    if len(data) > 0:
+        data['country_iso2'] = data.apply(lambda x: iso3_to_iso2(x['iso']), axis=1)
+    else:
+        return data
```

Comment:
The len lookup is not necessary, `apply` will return an empty series if called with and empty series. also, is better to apply the function to the desired column than to he whole dataframe and then look for the desired column. This could be simply:

```python
data['country_iso'] = data.iso.apply(iso3_to_iso2)
return data[column_order]
```

Reply:
Fixed